helm/idol-qms/README.md.gotmpl

```diff
@@ -0,0 +1,35 @@
+{{ template "chart.header" . }}
+{{ template "chart.deprecationWarning" . }}
+
+{{ template "chart.badgesSection" . }}
+
+{{- $component := "QMS" }}
+{{- $appVersion := .AppVersion }}
+{{- $versionList := slice (splitList "." $appVersion) 0 2 }}
+
+{{- $docsLink := printf "https://www.microfocus.com/documentation/idol/IDOL_%s/%s_%s_Documentation/Help/" (join "_" $versionList ) $component (join "." $versionList) }}
```

Comment:
```suggestion
{{- $docsLink := printf "https://www.microfocus.com/documentation/idol/IDOL_%s/%s_%s_Documentation/Help/" (join "." $versionList ) $component (join "." $versionList) }}
```
Docs link has apparently changed format for 24.1

Reply:
Can you change single-content/README.md.gotmpl accordingly too please?

Reply:
https://github.com/opentext-idol/idol-containers-toolkit/pull/27/commits/2166805112526881d7d50d48d54af2659ec8528f